torch/utils/data/dataloader.py

```diff
@@ -353,6 +364,9 @@ class DataLoader(object):
         worker_init_fn (callable, optional): If not None, this will be called on each
             worker subprocess with the worker id as input, after seeding and before data
             loading. (default: None)
+        ind_worker_queue (bool, optional): If ``True``, the worker will binded to invididual
+            processing queue. (default: False)
+
```

Comment:
Can you also add a `.. note:: ` on how to get deterministic results? Basically set ting`ind_worker_queue` and setting seed properly in `worker_init_fn`.

Reply:
Done.